### ISSUE TITLE
pre-defined noise

### ISSUE THREAD

**Issue by iperov** [juefeix/pnn.pytorch#5]
I read in the paper 

> When the input image comes in, it will be perturbed with a set of pre-defined random additive noise masks

Does this code always generate the same noise for each run for train and for evaluate?
```
if self.noise.numel() == 0:
            self.noise.resize_(x.data[0].shape).uniform_()
            self.noise = (2 * self.noise - 1) * self.level
```
